docs/network-flow-visibility.md

```diff
@@ -159,6 +160,8 @@ the flow. All the IEs used by the Antrea Flow Exporter are listed below:
 | ingressNetworkPolicyNamespace| 56506         | 111      | string      |
 | egressNetworkPolicyName      | 56506         | 112      | string      |
 | egressNetworkPolicyNamespace | 56506         | 113      | string      |
+| tcpState                     | 56506         | 136      | string      |
```

Comment:
As there are also some changes to the dashboard, do we need to add some info to the capabilities section in the doc?

Reply:
Actually on the dashboard side we only update fields in flow index pattern for Kibana to consume the new fields. No special visualization part is added.